software/myfin.yml

```
@@ -0,0 +1,37 @@
+---
+name: Addition
+about: Add new software to the list.
+title: Add MyFin
+labels: addition, reviewers wanted
+assignees: ''
+
+---
+
+Thanks for taking the time to suggest an addition to awesome-selfhosted! 
+
+Please fill out information below (all fields are mandatory unless noted otherwise):
+
+```yaml
+name: "MyFin"
+website_url: "https://github.com/afaneca/myfin"
+source_code_url: "https://github.com/afaneca/myfin"
+description: "Personal finances platform (web + rest api + android app) that'll help you budget, keep track of your income/spending and forecast your financial future."
```

```suggestion
description: "Personal finances platform (web + rest api + Android app) that'll help you budget, keep track of your income/spending and forecast your financial future."
```
How does it forecast your future?

---

By budgeting long-term, you can essentially forecast your financial future (how your patrimony will likely evolve over time. No witchcraft or complicated calculations involved, I swear :)

---

To build on the suggested above, I think this is cleaner and more accurate to sentence case 

```suggestion
description: "Personal finances platform (web + REST API + Android) that'll help you budget, keep track of your income/spending and forecast your financial future."
```